webdriver-spec.html

```diff
@@ -2397,14 +2397,10 @@
        will not be established.
 
      <dt><strong>Otherwise</strong>
-     <dd><p>If <var>name</var> is not the key of an
-      <a>extension capability</a> and the <a>remote end</a> is
-      an <a>endpoint node</a> return <a>success</a> with data
-      <code>null</code>.
-
-      <p>Let <var>value</var> be the result of <a>trying</a>
-       implementation-specific steps to match on <var>name</var>
-       with <var>value</var>. If the match is not
+       <dd><p>If <var>name</var> is the key of an
+      <a>extension capability</a>, <p>let <var>value</var> be the
+       result of <a>trying</a> implementation-specific steps to match
+       on <var>name</var> with <var>value</var>. If the match is not
```

Comment:
Fortuantely intermediary nodes never run this step since they all run the "return wahtever you  like as long as it has the right output" step.